site/src/theme/components/MuiButtonThemeOptions.ts

```diff
@@ -0,0 +1,303 @@
+        content: `""`,
+        borderRadius: "inherit",
+        position: "absolute",
+        width: "100%",
+        height: "100%",
+        border: "1px solid transparent",
+      };
+
+      const hoverStyles: CSSObject = {};
+
+      const disabledStyles: CSSObject = {};
+
+      const activeStyles: CSSObject = {};
+
+      const focusVisibleAfterStyles: CSSObject = {
+        content: `""`,
+        position: "absolute",
+        left: -buttonFocusBorderOffset,
+        top: -buttonFocusBorderOffset,
+        bottom: -buttonFocusBorderOffset,
+        right: -buttonFocusBorderOffset,
+        border: `${buttonFocusBorderWidth}px solid`,
+        borderRadius: 6 + buttonFocusBorderOffset,
+      };
+
+      if (variant === "primary") {
+        Object.assign(baseStyles, {
+          color: customColors.gray[20],
+          borderRadius: 34,
+          ...(size === "small" && {
+            padding: defaultTheme.spacing("8px", "20px"),
+          }),
+          ...(size === "medium" && {
+            padding: defaultTheme.spacing("12px", "28px"),
+            minHeight: 51,
+          }),
+          ...(color &&
+            {
+              purple: {
+                zIndex: 0,
+                background: customColors.purple[600],
+              },
+              teal: { backgroundColor: customColors.teal[500] },
+              gray: {
+                color: customColors.gray[80],
+                backgroundColor: customColors.gray[50],
+              },
+              warning: { backgroundColor: customColors.orange[500] },
+              danger: { backgroundColor: customColors.red[600] },
+              inherit: {},
+            }[color]),
+        });
+        Object.assign(beforeStyles, {
+          ...(color === "purple" && {
+            zIndex: -1,
+            opacity: 0,
+            transition: defaultTheme.transitions.create("opacity"),
+            background: `radial-gradient(57.38% 212.75% at 50.1% 134.31%, ${customColors.teal["400"]} 0%, ${customColors.purple[600]} 100%)`,
+          }),
+        });
+        Object.assign(hoverStyles, {
+          ...(color &&
+            {
+              purple: {
+                background: customColors.purple[600],
+                ":before": {
+                  opacity: 1,
+                },
+              },
+              teal: { backgroundColor: customColors.teal[600] },
+              gray: { backgroundColor: customColors.gray[30] },
+              warning: { backgroundColor: customColors.orange[600] },
+              danger: { backgroundColor: customColors.red[700] },
+              inherit: {},
+            }[color]),
+        });
+        Object.assign(disabledStyles, {
+          ...(color === "purple" && {
+            background: customColors.purple[200],
+            color: customColors.purple[100],
+          }),
+        });
+        Object.assign(focusVisibleAfterStyles, {
+          borderRadius: 34 + buttonFocusBorderOffset,
```

Comment:
We should probably make 34 a constant

Reply:
I think that's a good idea, but am not sure what to best call this as I did not implement it. @kachkaev could you please advice?

Reply:
Have called this `buttonBorderRadius` in this commit: https://github.com/blockprotocol/blockprotocol/pull/203/commits/a71b9f2d754a24c1af3ba6da170e650d4998c91a